lib/3lang/parser.md

```diff
@@ -0,0 +1,68 @@
+# 3bot parser
+
+- the objective of this parser is to be able to parse and execute actions from md files on different tf modules
+- we already have md file parser that could extract actions and their parameters.
+- these actions could be passed to the 3bot parser to parse and execute the user's requests
+- the 3bot parser should have smaller inner parsers for each tf module
+
+## Example
+
+- if a user wants to deploy 4 machines, it could go as follows:
+  
+```md
+    !!tfgrid.main.machines.deploy
```

Comment:
I'm not sure we want to have both main and helper. 3lang should be an easy to use intermediate language. So we will have to make decisions on behalf of the user to achieve this. Things like choosing the size of the resources, etc. What do you think @robvanmieghem?

Reply:
Agree, It should be simple to use and if this causes limitations, so be it.
Look at the example in the home story: https://circles.threefold.me/project/tf_engineering_coordination/us/91?kanban-status=2521

If a user wants to deploy 4 machines, I would suggest 4 `vm.create` actions are needed too.
I also suggest to make the gridselection a property and have 'main` as default instead of `tfgrid.main`